### ISSUE TITLE
[Bug]: Click to connect to the wallet, but nothing happens

### ISSUE THREAD

**Issue by king-168** [MetaMask/metamask-extension#15172]
### Describe the bug

点击连接钱包，没有任何反应啊，比如说opensea/chainlist网站，点击连接metamask钱包，钱包没有任何反应，也不连接网站，是什么原因呢？我把google恢复重置/重新启机/重新下载后会好一段，其后还链接不上网站，不知道官方有没有解决办法？

### Steps to reproduce

1.

### Version

10.16.2

### Browser

Chrome

### Operating system

Windows

**Comment by PeterYinusa** [comment]
**Translated the description using Google Translate:**
Click to connect to the wallet, but nothing happens. For example, on the opensea/chainlist website, click to connect to the metamask wallet, but the wallet does not respond and does not connect to the website. What is the reason? After I reset google/reboot/download it again, it will be good for a while, and then I can't link to the website. I don't know if there is an official solution?

**Comment by PeterYinusa** [comment]
Possibly related: #15142

**Comment by huoshan989** [comment]
这个bug我遇到两次了，不记得上次怎么弄好的，现在再次变成这样，很无奈

**Comment by Keepen** [comment]
> 这个bug我遇到两次了，不记得上次怎么弄好的，现在再次变成这样，很无奈

现在是2023年了，这个感觉还是有问题，我都把扩展给删了重新装，浏览器重开，点击【connect】还是没有任何响应

**Comment by eagle910** [comment]
我第一次遇到，刚才更新了，电脑也重启了，还是不行，uniswap又可以连接，pancake就不行

**Comment by PeterYinusa** [comment]
Please can you take a look at the following guide and let us know if this solves your issue: https://support.metamask.io/hc/en-us/articles/11907107168923

**Comment by PeterYinusa** [comment]
> 这个bug我遇到两次了，不记得上次怎么弄好的，现在再次变成这样，很无奈

Translated the comment using Google Translate:
I have encountered this bug twice, I don’t remember how to fix it last time, now it’s like this again, very helpless

> 现在是2023年了，这个感觉还是有问题，我都把扩展给删了重新装，浏览器重开，点击【connect】还是没有任何响应

Translated the comment using Google Translate:
It’s 2023 now, I still have a problem with this feeling, I deleted the extension and reinstalled it, reopened the browser, clicked [connect] and there was still no response

> 我第一次遇到，刚才更新了，电脑也重启了，还是不行，uniswap又可以连接，pancake就不行

Translated the comment using Google Translate:
This is the first time I encountered it. It was updated just now, and the computer was restarted. It still doesn’t work. Uniswap can be connected again, but pancake can’t.
